src/app.js

```diff
@@ -1,9 +1,8 @@
-/* eslint no-console: 0 */
-const express = require('express');
+const Server = require('./server');
 
-const app = express();
-const port = 8080;
+const instance = new Server({
+  port: 8080,
```

Comment:
Can we read this from `PORT` and default to 8080, cloud run inject a variable PORT

Reply:
Will do, thanks!